app/models/item.rb

```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
 class Item < ApplicationRecord
-  validates :type, presence: true
+  enum currency: {
+    usd: USD = 'USD',
+    eur: EUR = 'EUR',
+    uah: UAH = 'UAH'
+  }, _prefix: true
```

Comment:
why do we need prefix here?

Reply:
This prefix is more intuitive to use. For example, `item.currency_usd?` looks better for me instead of `item.usd?`